envoy/api/os_sys_calls.h

```diff
@@ -19,6 +19,9 @@ namespace Api {
 
 struct EnvoyTcpInfo {
   std::chrono::microseconds tcpi_rtt;
+  // Congestion window, in bytes. Note that posix's TCP_INFO socket option returns cwnd in packets,
+  // we multiply it by 1460(the default TCP MSS) to get bytes.
```

Comment:
1460 default MSS seems fine for internet traffic. I'm wondering if it worth to make it configurable for user controlled network or ENVOY_BUG if TCP MSS is not 1460. @mattklein123 for thoughts?